scripts/cloudcore_node.sh

```diff
@@ -20,13 +20,13 @@ if [ -f "${DIR}/prerequisite_packages.sh" ]; then
 source "${DIR}/prerequisite_packages.sh"
 fi
 
-rm -rf /root/.ssh/*
 key_gen (){
+   rm -rf /root/.ssh/id_rsa  &&  rm -rf /root/.ssh/id_rsa.pub
    echo y | apt-get update
    echo y | apt-get install sshpass
    < /dev/zero ssh-keygen -q -N ""
```

Comment:
See if just need do one time "ssh-keygen", so it cannot be mess up the " (/root/.ssh/id_rsa)" and " "/root/.ssh/id_rsa.pub"
if already have "/root/.ssh/id_rsa.pub", donot need generate key.